resources/options_lookup.tsv

```diff
@@ -1168,6 +1179,9 @@ HVAC System Heating Propane	"Propane Boiler, 76% AFUE"	ResidentialHVACBoiler	fue
 HVAC System Heating Propane	"Propane Boiler, 80% AFUE"	ResidentialHVACBoiler	fuel_type=propane	"system_type=hot water, forced draft"	afue=0.8	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																					
 		ResidentialAirflow	has_hvac_flue=true																																															
 		ResidentialHVACSizing	show_debug_info=false																																															
+HVAC System Heating Propane	"Propane Boiler, 82% AFUE"	ResidentialHVACBoiler	fuel_type=propane	"system_type=hot water, forced draft"	afue=0.82	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																					
+		ResidentialAirflow	has_hvac_flue=true																																															
+		ResidentialHVACSizing	show_debug_info=false																																															
 HVAC System Heating Propane	Propane Boiler	ResidentialHVACBoiler	fuel_type=propane	"system_type=hot water, forced draft"	afue=0.8	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																					

```

Comment:
I made that with a slightly different AFUE to meet the 2012 Federal Standard.

Reply:
Your option is fine. It's the one below it that is strange to me since there is no AFUE included in the name. I'd just want to remove it if it isn't being actively used anywhere.

Reply:
Got it. I confirmed `Propane Boiler` is not being used in any file in this repo or in `buildstockbatch`, so I removed the entry. Hooray cleanliness! 🚰